docsrc/acknowledgements/acknowledgements.html

```diff
@@ -1,19 +1,30 @@
+<div class="acklogo">
+	<a href="http://camptocamp.com" target="_blank"><img src="../_static/camptocamp.svg" style="max-width: 200px"
+		alt="Camptocamp" />
+	</a>
+	<p>
+		<a href="http://camptocamp.com" target="_blank">Camptocamp</a> provides development, support and hosting services for GeoNetwork Opensource solutions.</p>
+</div>
+<div class="acklogo">
+	<a href="http://geocat.net" target="_blank"><img src="../_static/GeoCat.png" alt="GeoCat bv" /></a>	 	
+	<p><a href="http://geocat.net" target="_blank">GeoCat bv</a> provides professional GeoNetwork development and hosting services.</p>
+</div>
 	<div class="acklogo">
- 	<a href="http://camptocamp.com"><img src="../_static/c2c.png" style="max-width: 200px"
-                                       alt="Camptocamp" /></a>
- 	<p><a href="http://camptocamp.com" target="_blank">Camptocamp</a> provides development, support and hosting services for GeoNetwork Opensource solutions.</p>
-	 </div>
-	<div class="acklogo">
- 	<a href="http://geocat.net"><img src="../_static/GeoCat.png" alt="GeoCat bv" /></a>	 	
- 	<p><a href="http://geocat.net" target="_blank">GeoCat bv</a> provides professional GeoNetwork development and hosting services.</p>
-	 </div>
-	 <div class="acklogo">
- 		<a href="http://www.jetbrains.com/idea/" style="position: relative;display:block; width:120px; height:30px; border:0; margin:0;padding:0;text-decoration:none;text-indent:0;" target="_blank"><span style="margin: 0;padding: 0;position: absolute;top: -1px;left: 29px;font-size: 9px;cursor:pointer;  background-image:none;border:0;color: #0d3a9e; font-family: trebuchet ms,arial,sans-serif;font-weight: normal;text-align:left;">Can’t code without</span><img src="http://www.jetbrains.com/idea/opensource/img/all/banners/idea120x30_white.gif" alt="The best Java IDE" border="0"/></a>
-	</div>
-	<div class="acklogo">
-		<a href="http://www.yourkit.com" target="_blank"><img src="../_static/YourKit-logo.png" alt="YourKit" /></a>            
-			<p>YourKit is kindly supporting open source projects with its full-featured Java Profiler. YourKit, LLC is the creator of innovative and intelligent tools for profiling Java and .NET applications.            <br/>Take a look at YourKit's leading software products:
-				<a href="http://www.yourkit.com/java/profiler/index.jsp" target="_blank">YourKit Java Profiler</a> and
-				<a href="http://www.yourkit.com/.net/profiler/index.jsp" target="_blank">YourKit .NET Profiler</a>.</p>
+		<a href="http://titellus.net" alt="titellus" target="_blank"><img src="../_static/titellus.png" width="135px" /></a>
+		<p>How to efficiently reference data and applying search technology to better discover and use information is the core of <a href="http://titellus.net" target="_blank">Titellus</a> expertise.
+		</p>
 	</div>
-	 
+<div class="acklogo">
```

Comment:
This logo can be removed, unfortunately JetBrains doesn't provide licenses anymore